modules/server/src/main/ts/BedrockAuto.ts

```diff
@@ -32,8 +32,10 @@ export const go = (bedrockAutoSettings: BedrockAutoSettings): void => {
 
   const routes = RunnerRoutes.generate('auto', settings.projectdir, settings.basedir, settings.config, settings.bundler, settings.testfiles, settings.chunk, settings.retries, settings.singleTimeout, settings.stopOnFailure, basePage, settings.coverage, settings.polyfills);
 
+  const shutdownServices: (() => Promise<any>)[] = [];
```

Comment:
Could these use `T` or are the services all random types?

Reply:
Changed to correct types: `Promise<void>`